tools/doc/html.js

```diff
@@ -446,7 +446,7 @@ function linkJsTypeDocs(text) {
     }
   }
 
-  //XXX maybe put more stuff here?
+  // XXX Maybe put more stuff here?
```

Comment:
Suggestion: being on it, you might also change `XXX` to `TODO`.

Reply:
Done.